ambry-frontend/src/main/java/com.github.ambry.frontend/AmbryBlobStorageService.java

```diff
@@ -492,12 +511,26 @@ public void onCompletion(String result, Exception exception) {
         try {
           RestMethod restMethod = restRequest.getRestMethod();
           logger.trace("Handling {} of {}", restMethod, result);
-          accountAndContainerInjector.injectTargetAccountAndContainerFromBlobId(result, restRequest);
+          BlobId blobId;
+          try {
```

Comment:
Since this try-catch code is used in multiple places, you can consider making a static util method